### ISSUE TITLE
Question about the model architecture (the batchnorm)

### ISSUE THREAD

**Issue by jihoontack** [Han-Jia/UNICORN-MAML#2]
Hi, again :)

Thank you for the wonderful work.

While reading your paper and seeing the implementation, it seems that this work used normal batchnorm (instead of transductive batchnorm that is usually used in other MAML-based papers)

Have you tried transductive batchnorm in your case? I was trying to use pre-trained checkpoints when training my MAML, but, I cannot reach the performance in your paper. I was just wondering whether the transductive batchnorm was the cause.

Thank you very much for your time!
Best,
Jihoon

**Comment by Han-Jia** [comment]
Hi, Jihoon,

We do not use transductive batchnorm in the paper, and we [reset the batchnorm](https://github.com/Han-Jia/UNICORN-zAML/blob/1681e4a72e7b240bb48b0dd66c993d3874919b9c/model/trainer/fsl_trainer.py#L201) for each task during the meta-test.

I think the learning rate and step size matter when using the pre-trained weights. A relatively larger learning rate and step size may help.

best,
Han-Jia

**Comment by jihoontack** [comment]
Thank you very much for your reply!

Really appreciate it :) 

Best,
Jihoon
